WalletWasabi.Fluent/Views/AddWalletPageView.axaml

```diff
@@ -6,65 +6,76 @@
              xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls"
              xmlns:conv="clr-namespace:WalletWasabi.Fluent.Converters"
              xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
+             xmlns:converters="clr-namespace:WalletWasabi.Fluent.Converters;assembly=WalletWasabi.Fluent"
              xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
              x:DataType="vm:HomePageViewModel"
              x:Class="WalletWasabi.Fluent.Views.AddWalletPageView">
-    <c:ContentArea Title="Add Wallet" Caption="Type in your wallet's name and select an option below:">
-        <DockPanel>
-            <StackPanel DockPanel.Dock="Top" Margin="0 0 0 40">
-                <Label Content="Wallet _Name:" Target="walletNameTb" />
-                <TextBox x:Name="walletNameTb" Text="{Binding WalletName}" Watermark="(e.g. My Wallet)">
-                    <i:Interaction.Behaviors>
-                        <behaviors:FocusOnAttachedBehavior />
-                    </i:Interaction.Behaviors>
-                </TextBox>
+  <UserControl.Resources>
+    <converters:BoolToOpacityConverter x:Key="BoolToOpacityConverter" />
+    <converters:OpacityToBoolConverter x:Key="OpacityToBoolConverter" />
+  </UserControl.Resources>
+
+  <c:ContentArea Title="Add Wallet" Caption="Type in your wallets name and select an option:">
+    <DockPanel>
+      <StackPanel DockPanel.Dock="Top" Margin="0 0 0 40">
+        <Label Content="Wallet _Name:" Target="walletNameTb" />
+        <TextBox x:Name="walletNameTb" Text="{Binding WalletName}" Watermark="(e.g. My Wallet)">
+          <i:Interaction.Behaviors>
+            <behaviors:FocusOnAttachedBehavior />
```

Comment:
is this okay @danwalmsley ? couldnt we have some property to just enable to make it a target of the focus first?

Reply:
how would it work?

This is fine for now... not sure there is a better way.

Reply:
oh okay, i thought we have something back in avalonia that could help with this

Reply:
IMO it is better to not touch the ViewModel with a property that is just changing the focus